### PR TITLE
Fix rhel8.10 image deploy issue.

### PR DESCRIPTION
**Pull request by Rajat-0**

### CODE REVIEW

**Comment by Rajat-0** [comment]
@SeanHQF @bjhuangr PTAL

File: zvmsdk/sdkwsgi/validation/parameter_types.py [hunk 1]
```diff
@@ -456,10 +456,10 @@ def valid_char(char):
 'oneOf': [
 {'type': 'string',
  'pattern':
- '^((r|R)(h|H)(e|E)(l|L))(6|7|8|9){1}([.][0-9])?$'},
+ '^((r|R)(h|H)(e|E)(l|L))(6|7|8|9){1}([.][0-9])?([0-9])*$'},
 {'type': 'string',
  'pattern':
- '^((r|R)(e|E)(d|D)(h|H)(a|A)(t|T))(6|7){1}([.][0-9])?$'},
+ '^((r|R)(e|E)(d|D)(h|H)(a|A)(t|T))(6|7){1}([.][0-9])?([0-9])*$'},
```
Comment (SeanHQF) [inline zvmsdk/sdkwsgi/validation/parameter_types.py:462]: what's this line used for?

File: zvmsdk/sdkwsgi/validation/parameter_types.py [hunk 2]
```diff
@@ -456,10 +456,10 @@ def valid_char(char):
 'oneOf': [
 {'type': 'string',
  'pattern':
- '^((r|R)(h|H)(e|E)(l|L))(6|7|8|9){1}([.][0-9])?$'},
+ '^((r|R)(h|H)(e|E)(l|L))(6|7|8|9){1}([.][0-9])?([0-9])*$'},
```
Comment (bjhuangr) [inline zvmsdk/sdkwsgi/validation/parameter_types.py:459]: maybe `'^((r|R)(h|H)(e|E)(l|L))(6|7|8|9){1}([.][0-9]{1,2})?$'` is better

File: zvmsdk/sdkwsgi/validation/parameter_types.py (hunk 1 above)
Comment (bjhuangr) [reply zvmsdk/sdkwsgi/validation/parameter_types.py:462]: `'^((r|R)(h|H)(e|E)(l|L))(6|7|8|9){1}([.][0-9]{1,2})?$'`

Comment (SeanHQF) [reply zvmsdk/sdkwsgi/validation/parameter_types.py:462]: I see, it's for redhatx.y

Comment (SeanHQF) [reply zvmsdk/sdkwsgi/validation/parameter_types.py:462]: So it supports both rhelx.y and redhatx.y

File: zvmsdk/sdkwsgi/validation/parameter_types.py (hunk 2 above)
Comment (Rajat-0) [reply zvmsdk/sdkwsgi/validation/parameter_types.py:459]: I'll test this

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**